IHP/IDE/CodeGen/MigrationGenerator.hs

```diff
@@ -214,7 +214,7 @@ migrateTable StatementCreateTable { unsafeGetCreateTable = targetTable } Stateme
                         (Just dropColumn) = actualColumns
                                 |> find \case
                                     Column { name } -> name == columnName
-                                    otherwise       -> False                                
+                                    otherwise       -> False
```

Comment:
Why was this changed?

Reply:
I'm not sure, it seems the editor reformatted it.